conduct/code-of-conduct.md

```diff
@@ -6,7 +6,7 @@ This is a template code of conduct that event organizers can use for conferences
 
```

Comment:
cc @karawoo @gdequeiroz

### Suggestion for adding more text on Values

I've taken the last sentence of the first paragraph and built on that for a sort of values paragraph. Thoughts on this?:

> We encourage the open exchange of ideas and expression and thus require an environment that recognizes the inherent worth of every person and group. An inclusive space free of harassment encourages interaction among diverse groups. `<EVENT>` is likely the first encounter of many people with this community; therefore, we want to make certain the experience is welcoming, and encourages them to be involved moving forward.

I had a hard time with this, so I thought I'd just get something down and then we can talk about it, and it seems important to have input from others rather than just my take